tests/integration/models/application.spec.coffee

```diff
@@ -23,16 +23,31 @@ describe 'Application Model', ->
 
 		describe 'balena.models.application.getAll()', ->
 
-			it 'should eventually become an empty array', ->
+			it 'should eventually become an empty array [Promise]', ->
 				promise = balena.models.application.getAll()
 				m.chai.expect(promise).to.become([])
 
+			it 'should eventually become an empty array [callback]', (done) ->
+				balena.models.application.getAll (err, applications) ->
+					m.chai.expect(err).to.be.null
+					m.chai.expect(applications).to.deep.equal([])
+					done()
+				return
```

Comment:
That's so that we are 100% sure that it was the `done` call that finished the test.